### ISSUE TITLE
Ingesting the NLP Highlights podcast

### ISSUE THREAD

**Issue by mjpost** [acl-org/acl-anthology#497]
The [NLP Highlights podcast](https://soundcloud.com/nlp-highlights) from AI2 has a wealth of material on a range of topics in NLP, from specific papers, to overviews of entire sub-fields, to career advice, and so on. I've listened to a number of these episodes and in addition to capturing a lot of interesting material, they are very well done. I got in touch with the hosts, Matt Gardner and Waleed Ammar, and AI2 is willing to archive this material in the Anthology, and also to contribute transcripts of the episodes.

I think this has lasting value and is worth preserving, but it presents two challenges that I'd like to solicit feedback on:

1. This is the first time we'll be ingesting non-peer-reviewed material
2. We currently only host papers as first-class objects

For (1), I don't think there is any issue, but it is probably worth discussing.

(2) is a technical issue related to how to represent this. NLP Highlights is currently not organized into anything like volumes. I have not given this too much thought. Perhaps we should have a second-level (beneath `<collection>`) `<podcast>` tag, with third-level `<episode>` tags serving as paper analogs? We could use `<editor>` for the hosts and `<author>` for whoever's being interviewed.

I think it would make sense to roll this out no sooner than EMLNP and no later than first quarter 2020, when we switch to the new Anthology ID format (#291).

Share your thoughts!

**Comment by nschneid** [comment]
"no later than first quarter 2020"—famous last words :) But let's revisit once we have the technical ability to host non-paper items like plenaries.

**Comment by mjpost** [comment]
It appears to be defunct: its original hosts have moved on and the last episode was February 29, 2024.

**Comment by nschneid** [comment]
Indeed, but it could still be good to archive the episodes that are there.

**Comment by mjpost** [comment]
Yes, I meant to imply that. It's all the more ready for archiving.
